Parsers/ASimRegistryEvent/Parsers/vimRegistryEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,150 @@
+Parser:
+  Title: Registry Event ASIM Parser for VMware Carbon Black Cloud
+  Version: '0.1.0'
+  LastUpdated: Oct 10, 2023
+Product:
+  Name: VMware Carbon Black Cloud
+Normalization:
+  Schema: RegistryEvent
+  Version: '0.1.2'
+References:
+- Title: ASIM Registry Schema
+  Link: https://aka.ms/ASimRegistryEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: VMware Carbon Black Cloud documentation 
+  Link: https://developer.carbonblack.com/reference/carbon-black-cloud/
+        https://developer.carbonblack.com/reference/cb-threathunter/latest/event-search-fields/
+Description: |
+  This ASIM parser supports normalizing VMware Carbon Black Cloud logs to the ASIM Registry Event normalized schema. VMware Carbon Black Cloud events are captured through VMware Carbon Black Cloud data connector which ingests Carbon Black Audit, Notification and Event data into Microsoft Sentinel through the REST API.
+ParserName: vimRegistryEventVMwareCarbonBlackCloud
+EquivalentBuiltInParser: _Im_RegistryEvent_VMwareCarbonBlackCloud
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: actorusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: registrykey_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: registryvalue_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: registryvaluedata_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvchostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventTypeLookup = datatable (temp_action: string, EventType: string)
+  [
+      "ACTION_WRITE_VALUE", "RegistryValueSet",
+      "ACTION_CREATE_KEY", "RegistryKeyCreated",
+      "ACTION_DELETE_KEY", "RegistryKeyDeleted",
+      "ACTION_DELETE_VALUE", "RegistryValueDeleted",
+      "ACTION_RENAME_KEY", "RegistryKeyRenamed"
+  ];
+  let RegistryKeyPrefixLookup = datatable(
+      RegistryKeyPrefix: string,
+      RegistryKeyNormalizedPrefix: string
+  )[
+      "HKLM", "HKEY_LOCAL_MACHINE",
+      "HKU", "HKEY_USERS",
+      "HKCU", "HKEY_CURRENT_USER",
+      "HKCR", "HKEY_CLASSES_ROOT",
+      "HKCC", "HKEY_CURRENT_CONFIG"
+  ];
+  let actionvalues = dynamic(["ACTION_WRITE_VALUE", "ACTION_CREATE_KEY", "ACTION_DELETE_KEY", "ACTION_DELETE_VALUE", "ACTION_RENAME_KEY"]);
+  let parser=(starttime: datetime=datetime(null), endtime: datetime=datetime(null), eventtype_in: dynamic=dynamic([]), actorusername_has_any: dynamic=dynamic([]), registrykey_has_any: dynamic=dynamic([]), registryvalue_has_any: dynamic=dynamic([]), registryvaluedata_has_any: dynamic=dynamic([]), dvchostname_has_any: dynamic=dynamic([]), disabled: bool=false) {
```

Comment:
please re-format. need to be in multiline

Reply:
We have already used Format-Query option but still it was in single line. So, we have manually changed it and make it multi-line.

Reply:
Please do manually in multi-line

Reply:
It's already done.